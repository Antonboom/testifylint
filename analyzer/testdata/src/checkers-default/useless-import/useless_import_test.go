// Code generated by testifylint/internal/testgen. DO NOT EDIT.

package uselessimport

import (
	"testing"

	_ "github.com/stretchr/testify/assert"  // want "useless-import: avoid the import of github.com/stretchr/testify/assert as _ since it doesn't do anything"
	_ "github.com/stretchr/testify/http"    // want "useless-import: avoid the import of github.com/stretchr/testify/http as _ since it doesn't do anything"
	_ "github.com/stretchr/testify/mock"    // want "useless-import: avoid the import of github.com/stretchr/testify/mock as _ since it doesn't do anything"
	_ "github.com/stretchr/testify/require" // want "useless-import: avoid the import of github.com/stretchr/testify/require as _ since it doesn't do anything"
	_ "github.com/stretchr/testify/suite"   // want "useless-import: avoid the import of github.com/stretchr/testify/suite as _ since it doesn't do anything"
)

func TestDummy(t *testing.T) {
	dummy := 1 + 3
	if dummy != 4 {
		t.Errorf("Dummy should return %d, but got %d", 4, dummy)
	}
}
