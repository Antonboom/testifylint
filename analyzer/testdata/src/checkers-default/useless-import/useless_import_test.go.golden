// Code generated by testifylint/internal/testgen. DO NOT EDIT.

package uselessimport

import (
	"testing"
)

func TestDummy(t *testing.T) {
	dummy := 1 + 3
	if dummy != 4 {
		t.Errorf("Dummy should return %d, but got %d", 4, dummy)
	}
}
